### ISSUE TITLE
TLS problem with no obvious reason

### ISSUE THREAD

**Issue by analyserdmz** [kgretzky/evilginx2#545]
Hello, could you please help me debug the root of the problem I am facing with Evilginx2 (latest)? It was working as expected with no problems and today I am getting a warning that says "Cannot write TLS response body from mitm'd client" with every test I do.

FYI the following are already checked and found to be correct:

- My certificates are valid, including every subdomain required by phishlets
- My domains are pointing to the correct IP address
- No firewalls, whatsoever
- I'm not blocked from the final destination services
- I am running evilginx as root

**Comment by emprrr** [comment]
bro I have the same problem, everything is set up as it should, did you solve this problem?

**Comment by analyserdmz** [comment]
No, unfortunately not yet. I hope somebody will answer the issue since it's here for almost half a year.

**Comment by emprrr** [comment]
you tried to use Modlishka
